packages/fast-development-site-react/src/components/site/component-wrapper.tsx

```diff
@@ -23,6 +23,7 @@ export interface IComponentWrapperProps<T> {
     singleRow?: boolean;
     designSystem: T;
     view: ComponentViewTypes;
+    dir: string;
```

Comment:
Also - we should do one of two things:

1. Make this optional. We use it, but by adding this as a required prop, we'll break anyone currently using the dev site that doesn't pass this prop
2. Pass a default if nothing is passed

I'm a fan of option 1. 

LTR is the default of the page itself, so if someone doesn't ever need to swap to rtl there is no reason for them to set dir here as the page is already in LTR

Reply:
It actually is, it should be exported from the fast-application-utilities package.

Reply:
The question to me isn't as much will it break someone. Rather 'should' this be required for production quality.  If the answer is yes, then it should be required.  We want to build tools that help others build quality sites without having to know all the ins and outs.

Reply:
I agree with the enum.  I also, think the `lang` attribute should be an enum.  But, may be out of scope here.

Reply:
By principle and policy, requiring this could constitute a breaking change and require a major version bump. Additionally, we can absolutely wire it in everywhere we want to use it, but we should not require this to others using this package. It's a feature that we light up, not functionality required. I would actually argue that it creates quality issues for people who do not need or want this functionality (IE, additional markup for "dir" attributes cluttered throughout the DOM with no actual purpose)